test/visibility/submodule-in-file/program-inline.chpl

```diff
@@ -0,0 +1,32 @@
+module Module {
```

Comment:
Is this one just here to see what the results are for a normal case / make sure that they match in terms of initialization order and such?

Reply:
yes